src/silo/config/database_config.cpp

```diff
@@ -44,4 +46,16 @@ ColumnType DatabaseMetadata::getColumnType() const {
    throw std::runtime_error("Unknown metadata type: " + std::string(name));
 }
 
+DatabaseMetadata DatabaseConfig::getMetadata(const std::string& name) const {
+   auto element = std::find_if(
+      std::begin(schema.metadata),
+      std::end(schema.metadata),
+      [&name](const auto& metadata) { return metadata.name == name; }
+   );
+   if (element == std::end(schema.metadata)) {
+      throw std::runtime_error("Unknown metadata type: " + std::string(name));
```

Comment:
I find type confusing, because seeing this as a user I would expect that I got the type STRING, PANGO_LINEAGE, INT, etc. wrong.
```suggestion
      throw std::runtime_error("Did not find metadata with name: " + std::string(name));
```